main/FatesRestartInterfaceMod.F90

```diff
@@ -1416,6 +1410,7 @@ subroutine create_patchcohort_structure(this, nc, nsites, sites, bc_in)
      use EDTypesMod,           only : ed_cohort_type
      use EDTypesMod,           only : ed_patch_type
      use EDTypesMod,           only : ncwd
+     use EDTypesMod,           only : maxSWb
      use EDTypesMod,           only : nlevleaf
```

Comment:
```suggestion
```

Reply:
note -- I'm suggesting we pull nlevleaf out of the restart infrastructure entirely as it should no longer be needed in this module.  is that correct?

Reply:
also there's a call to it in https://github.com/NGEET/fates/blob/44f66d60883d6b30affee95f97271fe498096add/main/FatesRestartInterfaceMod.F90#L1031 which i can't suggest removal of in the inline editor. could you delete it there too?

Reply:
done